.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ env:
   FUEL_CORE_PATCH_REVISION: ""
   RUST_VERSION: 1.85.0
   FORC_VERSION: 0.68.0
-  FORC_PATCH_BRANCH: ""
+  FORC_PATCH_BRANCH: "ironcev/error-codes-in-abi-json"
```

Comment:
Should this be present even post merge? I'd assume we don't want to have a patch branch on master, but don't have enough context to know for sure.

Reply:
Unfortunately we will have to release with this patch branch so that we unblock the sway team. Once they have a new release we will release again and remove the patch branch.